README.md

```diff
@@ -242,6 +244,22 @@ If you wish to use Uniauth with a project that already has users, a `UserProfile
 
 If your project does not fit either of these conditions, you will need to manually migrate the users as appropiate. Please create a `UserProfile` for each user, and `LinkedEmails` or `InstitutionAccounts` as appropiate.
 
+## Using JWT Authentication (API + UI split)
```

Comment:
⛏️ Let's add a link to this section in the Table of Contents

Reply:
Great! Also mention that [SessionMiddleware](https://github.com/lgoodridge/django-uniauth/blob/master/setup.py#L19) needs to be enabled, since we store the tokens in a session cookie.